### PR TITLE
Metalk8s storage split

### PR DESCRIPTION
**Pull request by Zempashi**

### CODE REVIEW

**Comment by NicolasT** [comment]
CI fails :)

File: roles/metalk8s_lvm_vg/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+- name: "LVM Setup: Detect old configuration style < 0.2.0"
```
Comment (ballot-scality) [inline roles/metalk8s_lvm_vg/tasks/main.yml]: This assertion doesn't belong to this place anymore

**Review by ballot-scality — DISMISSED** [review]
I'm OK with the changes and I need this PR to rework the tests and the check of the devices presence to avoid conflict

**Review by ballot-scality — APPROVED** [review]
Rebased on development/1.1

**Pull request merged**